client/app/layout.tsx

```diff
@@ -16,7 +17,16 @@ export default function RootLayout({
 }>) {
   return (
     <html lang='en'>
-      <body className={inter.className}>{children}</body>
+      <body className={inter.className}>
+        <main className='min-h-screen bg-amber-100'>
+          <Header />
+          {/* Figma design for desktop has 120px padding for both left and right sides, but feels too much, so set it to 56px*/}
+          <div className='px-4 pb-10 pt-8 md:px-14 md:pb-20 md:pt-10'>
```

Comment:
For Headers, we have 3 patters, which we can show with conditional state. So could you leave TODO comment for that? 
But for the `<div>` element, we can not set the padding for auth pages. So maybe it is better not to set here for now!

Reply:
I fixed it!
https://github.com/HAK2024/fairy-share/pull/55/commits/9c2e74a20abced66dca9035cdb1b7e6a9f96b88f